RGPosadas/LitterRally
# Task-70: Chat bubble list reusable component

This PR closes  #238
This PR aims to create a chat bubble list component that will be used to implement the announcement, direct message, and group chat pages. 


How to test:
1. Login into mull
2. click on the announcement page
3. you should see the following:

![image](https://user-images.githubusercontent.com/36641869/109408275-d3f28700-7955-11eb-9a96-7fffb46d6425.png)

If you would like to change the mocked data to test, you can add/delete posts, modify the data of the user here:
https://github.com/RGPosadas/Mull/pull/245/files#diff-013b1373bb1be74d0692f90c9faa273734f5cf86aeedceec0bb6c844a916de2cR11-R36
